tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/PLUS_drawn_215.txt

```diff
@@ -85,51 +85,51 @@
 0x2122  ™ trademark
 0x005E  ^ asciicircum
 0x007E  ~ asciitilde
-0x0300  ̀  gravecomb
+0x0300  ̀ gravecomb
```

Comment:
Why? the only thing I did was remove an extra space character before the glyph names. Which is a legacy from the original .nam files.